File: softnutri/src/main/java/br/com/softnutri/config/security/payload/response/JwtResponse.java
package br.com.softnutri.config.security.payload.response;

import java.util.Date;
import java.util.List;

import br.com.softnutri.records.ModuleDTO;

public record JwtResponse(String email, String token, String type, String refreshToken, String language, Date expiration,
		List<String> roles, List<ModuleDTO> modules) {

}

File: soft-nutri/src/app/model/user/jwt-response.ts
import { Injectable } from "@angular/core";
import { ModuleUser } from "../module/module";

@Injectable({
    providedIn: 'root',
  })
export class JwtResponse {
    public _token!: string;
    public refreshToken!: string;
    public type!: string;
    public roles!: string[];
    public language!: string;
    public expiration!: Date;
    public modules!: ModuleUser[];
}